sample-docker.env

```diff
@@ -0,0 +1,32 @@
+MONGOHQ_URL=mongodb://mongo:27017/freecodecamp
```

Comment:
I just wanted to make as simple as two commands, but perhaps I will just specify what needs to change in the original sample file.
- Remove the single quotes `'` in `MONGOHQ_URL`
- Replace `localhost` with `mongo` in `MONGOHQ_URL`